extensions/blocks/podcast-player/editor.scss

```diff
@@ -3,3 +3,9 @@
  */
 
 .wp-block-jetpack-podcast-player { }
+
+/* Back-compat for pre-G2 ToolbarButton elements containing text */
+.block-editor-block-contextual-toolbar[data-type="jetpack/podcast-player"] .components-toolbar__control,
+[data-type="jetpack/podcast-player"] .block-editor-block-contextual-toolbar .components-toolbar__control {
+	width: auto;
```

Comment:
Do you think we could add some extra padding to match the style of WP 5.4?

```suggestion
	width: auto;
	padding: 0 1em;
```

Otherwise the button seems a bit squeezed in 5.3.2:

![image](https://user-images.githubusercontent.com/426388/77062958-2a80e280-69dd-11ea-8434-cf0a9639ebd1.png)

The extra padding gets us to this:

**In WP 5.3.2**

![image](https://user-images.githubusercontent.com/426388/77063063-5bf9ae00-69dd-11ea-82fa-3ccabbeee30b.png)

**In WP 5.4**

![image](https://user-images.githubusercontent.com/426388/77063031-4b493800-69dd-11ea-8967-bacd03bb2240.png)

**with the latest version of Gutenberg**

![image](https://user-images.githubusercontent.com/426388/77063731-7da76500-69de-11ea-8892-e6c794ab6639.png)